mandatory/4-tax.js

```diff
@@ -17,8 +20,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let productPriceRes = calculateSalesTax(price);
```

Comment:
A really hard thing to do is choose variable names - Variable names should make it clear at a glance what is stored in the variable.

`Res` leaves it a little ambiguous and up to interpretation, partly because it is a shortened word. Perhaps something like `productPriceWithTax` would be clearer? What do you think?